SPECS/python-gevent/python-gevent.spec

```diff
@@ -48,6 +48,7 @@ Features include:
 %autosetup -p1 -n gevent-%{version}
 
 %build
+export GEVENTSETUP_DISABLE_ARES=1
```

Comment:
I can't speak, if this is the right way of fixing it, but we at least need an entry in the changelog.;)

Reply:
Oop! added :)